src/ParkChecklist/ParkChecklist.css

```diff
@@ -69,4 +69,30 @@
   .pagination button:hover {
     color: orange;
   }
+
+  .page-title{
+    position: relative;
+    margin-top: 0; 
+  }
+
+  .park-checklist-container {
+    position: relative;
+    min-height: 100vh; 
+    background-size: cover;
+    padding-top: 50px; 
+  }
+  
+  .loading-message {
```

Comment:
perfect, do you have anything you want me to hit on for this? ill take a look at WAVE to see how the contrast is

Reply:
I think it looks pretty good for now until we use Tailwind. But feel free to make changes if want. I'm not attached to any of the styling here. I'm just trying to make it presentable for this current stage of development.